Update Hebrew translation and some typos

Eng2Heb opened the pull request:
Header+strings

Razzeee commented:
Sorry, but as stated here https://github.com/trakt/script.trakt#translations we only accept translation via that way.

Eng2Heb commented:
Very good indeed.
I discovered on www.transifex.com the complete file and a missing translation in the Hebrew strings.po and even some typos & incorrect translation.
I fixed everything and added the missing in: https://www.transifex.com/teamxbmc/xbmc-addons/translate/#he/script-trakt/50421365
